iobio_tabs.js

```diff
@@ -0,0 +1,108 @@
+import { getDataBroker } from "./common.js";
+const tabsTemplate = document.createElement('template');
+tabsTemplate.innerHTML = `
+<style>
+:host {
+    --data-color: var(--iobio-data-color, #2d8fc1);
+}
+.iobio-tabs {
+    display: flex;
+    flex-direction: column;
+}
+.tabs {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+}
+.tab {
+    cursor: pointer;
+    text-align: center;
+    color: grey;
+}
+.tab.active {
+    color:  var(--data-color);
+}
+
+.content-container {
+    position: relative;
+    width: 100%;
+    height: 100%;
+}
+
+::slotted(iobio-histogram) {
```

Comment:
We don't want this to be dependent on the type of slotted element. We should be able to switch between any 2 HTML elements, even if they are different types.